src/main/kotlin/com/nexters/gamss/conversation/service/CommentGenerationService.kt

```
@@ -240,79 +243,43 @@ class CommentGenerationService(
+        ) { attempt ->
+            val output =
+                commentGenerator.generateReply(
+                    diaryContent,
+                    PromptCharacterId.of(characterMessage.emotionType!!).promptId,
+                    characterMessage.content,
+                    userReply,
                 )
-                return output
+            try {
+                commentFeedValidator.validateReply(output.text)
             } catch (e: CommentGenerationFailedException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)